test/test_image_scaling_bench.c

```diff
@@ -0,0 +1,62 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <LCUI_Build.h>
+#include <LCUI/LCUI.h>
+#include <LCUI/timer.h>
+#include <LCUI/display.h>
+#include <LCUI/gui/widget.h>
+#include <LCUI/gui/builder.h>
+#include <LCUI/graph.h>
+#include <LCUI/image.h>
+
+#ifdef LCUI_BUILD_IN_WIN32
+static void LoggerHandler(const char *str)
+{
+	OutputDebugStringA(str);
+}
+
+static void LoggerHandlerW(const wchar_t *str)
+{
+	OutputDebugStringW(str);
+}
+#endif
+
+int main(int argc, char **argv)
+{
+	LCUI_Graph test_image_scaling, test_image_scaling_result;
+	LCUI_Color t_color;
+	int i, t0, t1, t2;
+	int resx[] = { 480, 960, 1280, 1366, 1920, 2560, 3840 }, resy;
+#ifdef LCUI_BUILD_IN_WIN32
+	Logger_SetHandler(LoggerHandler);
+	Logger_SetHandlerW(LoggerHandlerW);
+#endif
+	Graph_Init(&test_image_scaling);
+	test_image_scaling.color_type = LCUI_COLOR_TYPE_RGB888;
+	if (Graph_Create(&test_image_scaling, 960, 540) < 0) {
+		return -2;
+	}
+	t_color.value = 0xffaa5500;
+	Graph_FillRect(&test_image_scaling, t_color, NULL, false);
+	LOG("%-20s%-20s%s\n", "image size\\method", "Graph_Zoom()",
+	    "Graph_ZoomBilinear()");
+	for (i = 0; i < sizeof(resx) / sizeof(int); i++) {
+		resy = resx[i] * 9 / 16;
+		t0 = LCUI_GetTime();
+		Graph_Init(&test_image_scaling_result);
+		Graph_Zoom(&test_image_scaling, &test_image_scaling_result,
+			   false, resx[i], resy);
+		Graph_Free(&test_image_scaling_result);
+		t1 = LCUI_GetTime();
+		Graph_Init(&test_image_scaling_result);
+		Graph_ZoomBilinear(&test_image_scaling,
+				   &test_image_scaling_result, false, resx[i],
+				   resy);
+		Graph_Free(&test_image_scaling_result);
+		t2 = LCUI_GetTime();
+		LOG("%dx%-20d%d%-20s%d%s\n", resx[i], resy, t1 - t0, "ms",
```

Comment:
The result is not aligned to the left:

```
./test_image_scaling_bench
image size\method   Graph_Zoom()        Graph_ZoomBilinear()
480x270                 1ms                  28ms
960x540                 6ms                  95ms
1280x720                 13ms                  160ms
1366x768                 16ms                  185ms
1920x1080                24ms                  368ms
2560x1440                42ms                  698ms
3840x2160                109ms                  1457ms
```

Example of improvements that can be referenced:

```
char size_str[32], time1_str[32], time2_str[32];

sprintf(str_size, "%dx%d", resx[i], resy):
sprintf(time1_str, "%dms", t1 - t0);
sprintf(time2_str, "%dms", t1 - t0);
LOG("%-20s%-20s%-20s\n", size_str, time1_str, time2_str);
```